Library/ViewModels/PledgeViewModel.swift

```diff
@@ -338,6 +361,70 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     let createBackingTransactionSuccess = project.takeWhen(createBackingEventSuccess)
 
     self.goToThanks = Signal.merge(applePayTransactionCompleted, createBackingTransactionSuccess)
+
+    let amountChanged = Signal.combineLatest(
+      project.map { $0.personalization.backing?.pledgeAmount },
+      self.pledgeAmountDataSignal.map { $0.0 }
```

Comment:
`self.pledgeAmountDataSignal.map(first)`? Might also be valuable to unpack the backing seperately, for clarity:

```
let backing = project.map { $0.personalization.backing }.skipNil()

let pledgeAmount = Signal.combineLatest(
  backing.map { $0.pledgeAmount }, 
  self.pledgeAmountDataSignal.map(first))
```

Looks like `backing` would also come in useful below in a couple places.

Reply:
Can't do `self.pledgeAmountDataSignal.map(first)` because the tuple is 4 values and our versions of `first`, `second`, `third` only support up to 3.

Reply:
Alas